style_guide.md

```diff
@@ -0,0 +1,304 @@
+    local questionGuidance(isProxy) = (
+      if isProxy == true then 'Why we ask this question if they are retired or long-term sick or disabled' else 'Why we ask this question if you are retired or long-term sick or disabled';
+    );
+    ```
+#### Multiple value variants example
+
+- There are occasions where more than one variable is used to create variants.
+- Example of this would be a `Question` block using combination of `isProxy` and `isEmployed` variables:
+
+    ```
+    {
+      type: 'Question',
+      id: 'business-name',
+      question_variants: [
+        {
+          question: question(isProxy=false, isEmployed=true),
+          when: [rules.isNotProxy, rules.isEmployed],
+        },
+        {
+          question: question(isProxy=true, isEmployed=true),
+          when: [rules.isProxy, rules.isEmployed],
+        },
+        {
+          question: question(isProxy=false, isEmployed=false),
+          when: [rules.isNotProxy, rules.isNotEmployed],
+        },
+        {
+          question: question(isProxy=true, isEmployed=false),
+          when: [rules.isProxy, rules.isNotEmployed],
+        },
+      ],
+    }
+    ```
+- Both variables are passed to `local question` used by `Question` method:
+    ```
+    local question(isProxy, isEmployed) = {
+      id: 'business-name-question',
+      title: questionTitle(isProxy, isEmployed),
+      description: description,
+      type: 'MutuallyExclusive',
+      mandatory: false,
+      answers: [
+        {
+          id: 'business-name-answer',
+          label: 'Organisation or business name',
+          max_length: 160,
+          mandatory: false,
+          type: 'TextField',
+        },
+        {
+          id: 'no-business-name-answer',
+          mandatory: false,
+          type: 'Checkbox',
+          options: [
+            {
+              label: option,
+              value: option,
+            },
+          ],
+        },
+      ],
+    };
+    ```
+- To resolve title, we call a new questionTitle method, passing `isProxy` and `isEmployed`:
+    ```
+    local questionTitle(isProxy, isEmployed) = (
+      if isProxy == true && isEmployed == true then {
+      text: 'What is the main activity of <em>{person_name_possessive}</em> organisation, business or freelance work?',
+      placeholders: [
+        placeholders.personNamePossessive,
+      ]}
+      else if isProxy == false && isEmployed == true then {
+      text: 'What is the main activity of <em>{person_name_possessive}</em> organisation, business or freelance work?',
+      placeholders: [
+        placeholders.personNamePossessive,
+      ]}
+      else if isProxy == true && isEmployed == false then 'What was the main activity of your organisation, business or freelance work?'
+      else {
+      text: 'What was the main activity of <em>{person_name_possessive}</em> organisation, business or freelance work?',
+      placeholders: [
+        placeholders.personNamePossessive,
+      ]};
+    );  
```

Comment:
This should be structured in a similar way to my comment about the single value example.

Reply:
I put it in single block, but left guidance example below if you don't mind, I think it is useful.